eva_submission/eload_brokering.py

```diff
@@ -39,7 +39,11 @@ def broker(self, brokering_tasks_to_force=None, existing_project=None):
         self.broker_to_ena(force=('ena' in brokering_tasks_to_force), existing_project=existing_project)
 
     def prepare_brokering(self, force=False):
-        if not self.eload_cfg.query('brokering', 'analyses', 'vcf_files') or force:
+        valid_analyses = self.eload_cfg.query('validation', 'valid', 'analyses', [])
```

Comment:
```suggestion
        valid_analyses = self.eload_cfg.query('validation', 'valid', 'analyses', ret_default=[])
```
(I think otherwise the `*args` eats the last argument...)